builtin/providers/aws/resource_aws_cloudwatch_metric_alarm.go

```diff
@@ -95,6 +96,12 @@ func resourceAwsCloudWatchMetricAlarm() *schema.Resource {
 				Optional:      true,
 				ConflictsWith: []string{"statistic"},
 			},
+			"treat_missing_data": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
I'm wondering if `Default: "missing"` would be more suitable here 🤔  That would make people put that field if it drifts from the default value, rather than leave it out and let anyone mess with that outside of Terraform.